source/gloperate/include/gloperate/base/TimeManager.h

```diff
@@ -154,17 +158,17 @@ class GLOPERATE_API TimeManager : public cppexpose::Object
 
 
 protected:
-    Environment                * m_environment;   ///< Gloperate environment to which the manager belongs
+    Environment                               * m_environment;   ///< Gloperate environment to which the manager belongs
 
     // General timers
-    unsigned int                 m_activeTimers;  ///< Number of active timers
-    std::vector<Timer *>         m_timers;        ///< List of registered timers
-    gloperate::ChronoTimer       m_clock;         ///< Time measurement
-    float                        m_timeDelta;     ///< Time delta since last update (in seconds)
+    unsigned int                                m_activeTimers;  ///< Number of active timers
+    std::vector<Timer *>                        m_timers;        ///< List of registered timers
```

Comment:
`registerTimer` is called only once in the `Timer` constructor. So the code creating a new Timer owns the object. e.g. AutoTimer, DemoStage, TimerStage, and TimeManager (in m_scriptTimers)